### PR TITLE
base colors and typography

### PR DESCRIPTION
**Pull request by pflopez**

### CODE REVIEW

File: src/interface/src/styles/typography.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
+// base
+body {
+  font-family: Public Sans, sans-serif;
```
Comment (pflopez) [inline src/interface/src/styles/typography.scss:3]: this is being applied, but we are overriding all over the place with roboto, so I dont think it ever gets to use this

File: src/interface/src/styles/typography.scss [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+.p-important {
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 32px;
+}
+
+
+// standard link
+.a {
```
Comment (pflopez) [inline src/interface/src/styles/typography.scss:68]: we already had a class `.link` which was conflicting with this one

File: src/interface/src/styles/typography.scss (hunk 1 above)
Comment (lastminutediorama) [reply src/interface/src/styles/typography.scss:3]: I also discovered this when I tried to change fonts in `body`, but maybe this is okay for now.
Should we have a convention to quote multi-word font names, or no?

**Review by lastminutediorama — APPROVED** [review]
This is a nicer approach than what I was thinking, thanks for adding it.

**Pull request closed without merging**